.github/workflows/argilla-sdk.yml

```diff
@@ -132,5 +132,5 @@ jobs:
           pip install --index-url https://test.pypi.org/simple --no-deps  $PACKAGE_NAME==$PACKAGE_VERSION
 
       - name: Publish Package to PyPI 🥩
-        if: ${{ inputs.release }}
+        if: ${{ inputs.release == 'true' }}
```

Comment:
This change is because you are expecting to run the action using `gh` cli manually?

Reply:
the workflow will be run manually yes, but you don't need the gh CLI. You can run it through the web https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow#running-a-workflow

<img width="431" alt="Captura de pantalla 2024-05-23 a las 22 54 37" src="https://github.com/argilla-io/argilla-python/assets/2518789/745fbf99-15f1-4607-983f-fecdb6d5d6e2">